hudi-client/hudi-client-common/src/main/java/org/apache/hudi/client/transaction/lock/StorageBasedLockProvider.java

```diff
@@ -193,7 +196,7 @@ public boolean tryLock(long time, TimeUnit unit) {
         if (tryLock()) {
           return true;
         }
-        Thread.sleep(DEFAULT_LOCK_ACQUISITION_BUFFER_MS);
+        Thread.sleep(Long.parseLong(DEFAULT_LOCK_ACQUIRE_RETRY_WAIT_TIME_IN_MILLIS));
```

Comment:
again, trying to think of the necesity of this retry. we do have inherent retries w/ LockManager which will kick in every configured value of LOCK_ACQUIRE_CLIENT_RETRY_WAIT_TIME_IN_MILLIS_PROP_KEY 
anyways. So, can you help explain the necessity of this? 
also, if we have this, likely the retry from LockManager might be ineffective? 

or can you explain why do we need retries in both layers?

Reply:
I can get rid of the retries here too.

Reply:
Ok I actually re-reviewed the code and this follows the same pattern as dynamodblockclient. Which has a default buffer of 1000s and sleeps each time it is unable to acquire the lock. Adding screenshots below

Reply:
![image](https://github.com/user-attachments/assets/54068a8e-9663-43f0-b075-73d2a347ed3d)
![image](https://github.com/user-attachments/assets/4d64a2e5-5153-41b2-b88f-907eb666b76c)
![image](https://github.com/user-attachments/assets/a23ac115-d49f-4114-bd6b-fe48dd5db819)